book/page_creation.rst

```diff
@@ -73,6 +73,12 @@ Before diving into this, test it out!
     replace ``http://localhost:8000`` with your host name - like
     ``http://symfony.dev/app_dev.php/lucky/number``.
 
+.. tip::
+    if you cant see your page try to clear cache using command
+    ``php bin/console  cache:clear --env=prod``
```

Comment:
It looks strange that clearing the cache for the `prod` environment should help with your `dev` environment.